app/scripts/views/sign_up.js

```diff
@@ -142,6 +142,7 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       return {
         serviceName: relier.get('serviceName'),
         isSync: relier.isSync(),
+        isChooseWhatToSyncWeb: this._isSyncChooser(),
```

Comment:
Maybe call this variable `shouldShowChooseWhatToSync` or just `showChooseWhatToSync`?